README.md

```diff
@@ -89,6 +89,10 @@ const style = css({
 })
 ```
 
+### Images and other static files
+
+Create a folder called `static` in your project root directory. You can place images and other static assets there. Reference them from JS files with the path to the static directory. For example, from `/pages` use `../static/file-name.jpg`
```

Comment:
oh, it's because the image is in a component in `./components/`. I should change it with two examples...